### PR TITLE
166789837 Article star rating

### PR DESCRIPTION
**Pull request by Emile-Nsengimana**
#### What does this PR do?
This PR will create UI component to:
- View articles
- Rate an article

#### Description of Task to be completed?
A 5 star rating system is the end goal. Lets have authenticated users get a visual feedback of their average rating of their articles
#### How should this be manually tested?
First `git clone` https://github.com/andela/ah-lobos-frontend.git

Then :
navigate into the project directory
>cd ah-lobos-frontend 

 install all dependencies

>npm install

run test for end-to-end

>npm run cypress

run test for unit test

>npm run test

start application

>npm start

- Your default browser will open, click ```read``` on any article then ```rate``` the article with the number of ```star``` you prefer(1-5) on the bottom of the article

#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#166789837](https://www.pivotaltracker.com/story/show/166789837)
###### 1. 
rate

###### 2. 
Rate_1

#### Questions:
- N/A

### CODE REVIEW

**Review by manzif — COMMENTED** [review]
src/styles/mains.scss

Good work!! But you have commented codes in that file. Please correct it.

**Comment by Emile-Nsengimana** [comment]
> src/styles/mains.scss
> 
> Good work!! But you have commented codes in that file. Please correct it.

Thanks, Good eye I have now removed unnecessary `css style`.

**Review by manzif — APPROVED** [review]

File: src/assets/images/follow.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg id="Component_2_1" data-name="Component 2 – 1" xmlns="http://www.w3.org/2000/svg" width="78.06" height="23.995" viewBox="0 0 78.06 23.995">
+  <path id="ic_add_circle_24px" d="M14,2A12,12,0,1,0,26,14,12,12,0,0,0,14,2Zm6,13.2H15.2V20H12.8V15.2H8V12.8h4.8V8h2.4v4.8H20Z" transform="translate(52.065 -2)" fill="#0dc68d"/>
+  <text id="Follow" transform="translate(22 17.318)" fill="#0dc68d" font-size="15" font-family="Helvetica"><tspan x="-21.672" y="0">Follow</tspan></text>
+</svg>
```
Comment (adafia) [inline src/assets/images/follow.svg]: Please add a blank line at the end of this file

**Review by adafia — APPROVED** [review]

File: src/assets/images/follow.svg (hunk 1 above)
Comment (Emile-Nsengimana) [reply src/assets/images/follow.svg]: It is now done.

**Pull request merged**